Integrations/integration-QRadar.yml

```diff
@@ -235,13 +235,13 @@ script:
             var totalOffenses = res.length;
             var lastCallOffenses = res.length;
             while (startTime && lastCallOffenses >= offensesPerCall) {
-                var min = res.length;
-                var max = min + offensesPerCall;
+                var from = totalOffenses + 1;
+                var to = min + offensesPerCall;
```

Comment:
@omercnet does the new code work? it seems in this line that we are calling the variable `min`, but it doesn't exist anymore...

Reply:
thanks! fixed